src/backend/LosTomates.PetHolidays.Application/Pets/PetView.cs

```diff
@@ -0,0 +1,10 @@
+﻿using LosTomates.PetHolidays.Application.PetTypes;
+
+namespace LosTomates.PetHolidays.Application.Pets;
+
+public class PetView
+{
+    public int Id { get; set; }
+    public string? Name { get; set; }
+    public PetTypeView? PetType { get; set; }
```

Comment:
Так как у нас всегда будет он заполнен, то можно убрать признак nullable здесь, вот так сделать:
```suggestion
    public required PetTypeView PetType { get; set; }
```